metecho/api/views.py

```diff
@@ -191,8 +191,9 @@ def disconnect(self, request):
     @extend_schema(request=None, responses={202: None})
     @action(methods=["POST"], detail=False)
     def refresh(self, request):
-        """Queue a job to refresh the user's list of GitHub repositories."""
+        """Queue a job to refresh the user's list of GitHub repositories and organizations."""
         request.user.queue_refresh_repositories()
+        GitHubOrganization.queue_get_orgs_for_user(request.user)
```

Comment:
@jerivas I think this should likely move to the `User.queue_refresh_repositories` method, so that it is also called when repos are [refreshed upon login](https://github.com/oddbird/Metecho/blob/new-project/metecho/api/models.py#L1578-L1580).

Reply:
Fixed in b35ab2ade6bbd8080405f27e40865145a810519e